beacon-chain/node/node.go

```diff
@@ -1043,13 +1042,13 @@ func (b *BeaconNode) registerPrometheusService(_ *cli.Context) error {
 	return b.services.RegisterService(service)
 }
 
-func (b *BeaconNode) registerHTTPService(router *mux.Router) error {
+func (b *BeaconNode) registerHTTPService(router *middleware.NormalizeQueryValuesHandler) error {
 	host := b.cliCtx.String(flags.HTTPServerHost.Name)
 	port := b.cliCtx.Int(flags.HTTPServerPort.Name)
 	address := net.JoinHostPort(host, strconv.Itoa(port))
 	timeout := b.cliCtx.Int(cmd.ApiTimeoutFlag.Name)
 	opts := []httprest.Option{
-		httprest.WithRouter(router),
+		httprest.WithRouter(router.Router),
```

Comment:
This doesn't work, you are just returning the underlying servemux, that's not wrapped ...